lib/tasks/temp.rake

```diff
@@ -495,7 +495,7 @@ namespace :temp do
           path = helper.get_attachment_as_html_url(params)
           protocol = AlaveteliConfiguration.force_ssl ? 'https' : 'http'
           domain = AlaveteliConfiguration.domain
-          url = "#{ protocol }://#{ domain }#{ path }"
+          url = "#{protocol}://#{domain}#{path}?skip_cache=1"
```

Comment:
I _think_ we should generate a random value for the `skip_cache` param.

Since we're hitting the app externally, the error page might get cached by varnish, so if you then run it again you'll always get a cached response?

Reply:
I've done this a rebased.